ec2utils/ec2deprecateimg/MANIFEST.in

```diff
@@ -0,0 +1,8 @@
+include LICENSE
+include Makefile
+include python3-ec2deprecateimg.spec
+include README.md
+
+recursive-include man *.1
+recursive-exclude * __pycache__
+recursive-exclude * *.py[co]
```

Comment:
Wit the creation of MANIFEST.in the pypi target in the Makefile should no longer be needed, please verify that the tarball contains everything and then remove the target in the Makefile.

Reply:
All PR's updated and tarballs confirmed.